### ISSUE TITLE
How can I apply the equation to make binary classification

### ISSUE THREAD

**Issue by hickskin** [trevorstephens/gplearn#226]
I have a question about SymbolicClassifier. What is the meaning of resulting formulae. For example, `x2/x1 + x3- x5`. How can I apply the equation to make binary classification ? My oal is to make binary classification on my data.

**Comment by IFFranciscoME** [comment]
you might consider using the output of the equation as an input to an activation function, a sigmoid for example will therefore create a logistic regression classifier.

**Comment by hickskin** [comment]
Thank you for your response. Since it is a classifier, can not we learn the model to find a threshold value for the output equation so that it makes binary classification ?

**Comment by trevorstephens** [comment]
https://gplearn.readthedocs.io/en/stable/intro.html#classification a sigmoid is used by default to transform the output

**Comment by hickskin** [comment]
yes sigmoid has been used to classify the data. However, my requirement is to get equation from `_Program` after the sigmoid is applied. What is printing now is before the sigmoid  is applied.

**Comment by trevorstephens** [comment]
Yes, it's possible to add the sigmoid to the representation printed on the screen. I can put this in the long-overdue next release.

**Comment by trevorstephens** [comment]
I think it makes sense to do it, thanks for the report
